src/rcdash_setup_template

```diff
@@ -1,12 +1,12 @@
-#!/bin/bash -e
```

Comment:
This was done because if whiptail exit with a non-zero exit status it would just kill the entire script.  I need to catch it's output in same cases.